packages/shared/src/hooks/useChangelog.ts

```diff
@@ -0,0 +1,59 @@
+import { useContext, useMemo } from 'react';
+import { useQuery } from 'react-query';
+import AlertContext from '../contexts/AlertContext';
+import AuthContext from '../contexts/AuthContext';
+import { getLatestChangelogPost, Post } from '../graphql/posts';
+import { laptop } from '../styles/media';
+import useMedia from './useMedia';
+
+type UseChangelog = {
+  isAvailable: boolean;
+  latestPost?: Post;
+  dismiss: () => Promise<void>;
+};
+
+export function useChangelog(): UseChangelog {
+  const { alerts, updateAlerts } = useContext(AlertContext);
+  const { user } = useContext(AuthContext);
+  const isDesktop = useMedia([laptop.replace('@media ', '')], [true], false);
```

Comment:
We have an existing hook called `useSidebarRendered`, it returns a property named `sidebarRendered` with a boolean value. If it is true, then it is desktop. We can probably utilize it.